.pre-commit-config.yaml

```diff
@@ -1,18 +1,19 @@
 repos:
-  - repo: https://github.com/asottile/seed-isort-config
-    rev: v2.2.0
-    hooks:
-      - id: seed-isort-config
-  - repo: https://github.com/pre-commit/mirrors-isort
-    rev: v5.9.1
-    hooks:
-      - id: isort
-  - repo: https://github.com/psf/black
-    rev: 21.6b0
-    hooks:
-      - id: black
-        language_version: python3
-  - repo: https://github.com/pycqa/flake8
-    rev: 3.9.2
-    hooks:
-      - id: flake8
+    - repo: https://github.com/PyCQA/isort
+      rev: 5.9.3
+      hooks:
+          - id: isort
+            name: isort (python)
+    - repo: https://github.com/psf/black
+      rev: 21.6b0
+      hooks:
+          - id: black
+            language_version: python3
+    - repo: https://github.com/pycqa/flake8
+      rev: 3.9.2
+      hooks:
+          - id: flake8
+    - repo: https://github.com/pre-commit/mirrors-prettier
```

Comment:
Not sure why we need prettier when we have `black`?

Reply:
Ah, I think to handle the markdown formatting? Can we tell it to ignore python files in that case? I am concerned that we will end up with black and prettier fighting each other.

Reply:
Nice ok.